status-payloads-spec.md

```diff
@@ -65,46 +65,100 @@ If a signature is not present but an author is provided by a layer below, the me
 
 ## Encoding
 
-The payload is encoded using [Transit format](https://github.com/cognitect/transit-format). Transit was chosen over JSON in order to reduce the bandwidth.
+The payload is encoded using [Protobuf](https://developers.google.com/protocol-buffers)
 
 ## Types of messages
 
 ### Message
 
-The type `Message` represents a text message exchanged between clients and is identified by the transit tag `c4`.
+The type `ChatMessage` represents a text message exchanged between clients.
 
 #### Payload
 
-Payload is a struct (a compound data type) with the following fields (order is important):
+The protobuf description is:
 
-<!-- TODO: Be more precise in struct description, a la RFC, e.g. TLS style https://tools.ietf.org/html/rfc8446 -->
+```protobuf
+message ChatMessage {
+  // Lamport timestamp of the chat message
+  uint64 clock = 1;
+  // Unix timestamps in milliseconds, currently not used as we use whisper as more reliable, but here
```

Comment:
I think I know what you mean, but this can probably be rephrased for clarity

Reply:
Not sure it's actually even worth mentioning is not used, the only way we use it is for cosmetic (to show a user what time a message has been sent, but it's purely cosmetic (another client might chose not to do so), what do you think?